lib/node_modules/@stdlib/blas/ext/base/dsumors/test/test.dsumors.js

```diff
@@ -95,9 +94,9 @@ tape( 'if provided an `N` parameter equal to `1`, the function returns the first
 });
 
 tape( 'the function supports a `stride` parameter', function test( t ) {
-	var N;
 	var x;
 	var v;
+	var N;
```

Comment:
Please remove the now unused `N` variable declarations in this and the other test files.